ansible/roles/schulcloud-server-core/templates/preview-generator-deployment.yml.j2

```
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: preview-generator-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: preview-generator
+spec:
+  replicas: {{ AMQP_FILE_PREVIEW_REPLICAS|default("1", true) }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      #maxUnavailable: 1
+  revisionHistoryLimit: 4
+  paused: false
+  selector:
+    matchLabels:
+      app: preview-generator
+  template:
+    metadata:
+      labels:
+        app: preview-generator
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+        runAsNonRoot: true
+      containers:
+      - name: preview-generator
+        image: {{ SCHULCLOUD_SERVER_IMAGE }}:file-preview-{{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+        imagePullPolicy: IfNotPresent
+        envFrom:
+        - configMapRef:
+            name: preview-generator-configmap
+        - secretRef:
+            name: preview-generator-secret
+        command: ['npm', 'run', 'nest:start:preview-generator-amqp:prod']
+        resources:
+          limits:
+            cpu: {{ AMQP_FILE_PREVIEW_CPU_LIMITS|default("1000m", true) }}
+            memory: {{ AMQP_FILE_PREVIEW_MEMORY_LIMITS|default("2500Mi", true) }}
+          requests:
+            cpu: {{ AMQP_FILE_PREVIEW_CPU_REQUESTS|default("100m", true) }}
+            memory: {{ AMQP_FILE_PREVIEW_MEMORY_REQUESTS|default("50Mi", true) }}
```

So little adjustments to the default value for memory and cpu
```suggestion
            cpu: {{ AMQP_FILE_PREVIEW_CPU_LIMITS|default("4000m", true) }}
            memory: {{ AMQP_FILE_PREVIEW_MEMORY_LIMITS|default("4000Mi", true) }}
          requests:
            cpu: {{ AMQP_FILE_PREVIEW_CPU_REQUESTS|default("100m", true) }}
            memory: {{ AMQP_FILE_PREVIEW_MEMORY_REQUESTS|default("250Mi", true) }}
```